All Core Devs Meetings/meeting 94.md

```diff
@@ -0,0 +1,437 @@
+# All Core Devs Meeting 94 Notes
+### Meeting Date/Time: Friday, Aug 21 2020, 14:00 UTC
+### Meeting Duration: 1:32 hrs
+### [GitHub Agenda](https://github.com/ethereum/pm/issues/200)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=q6bIeSu7r9o&t=2835s)
+### Moderator: Hudson Jameson
+### Notes: Alita Moore
+  
+---
+  
+# Summary
+  
+## EIP Status
+EIP | Status
+  
+  
+## Decisions Made
+  
+Decision Item | Description
+--|--
+**94.1.1**: Discuss the possibility of launching a Prog POW on a test net next week in line with the Ben's compromise
```

Comment:
Formatting

**94.1.1**| Discuss the possibility of launching a Prog POW on a test net next week in line with the Ben's compromise